### DISCUSSION TITLE
Wake up from sleep mode in automation

### DISCUSSION THREAD

**Discussion by andre1210** [Blackymas/NSPanel_HA_Blueprint#1826] [Q&A] [answered]
Hi,
in earlier versions (I don't remember the version) there was a switch for sleep mode. I had an automation that switched to sleep mode between 22:00 and 6:00. That worked - in the morning, the display was illuminated automatically.
In the current version (4.2.6) this switch isn't available any more.
In my automation I set at 22:00 the sleep timeout to 10sec and the display turns dark.
At 06:00 I set the timeout to 0sec, but the display doesn't turn on automatically. I have to touch it once and it stays on.
What do I wrong, or what could be a workaround? How can it wake up automatically?
Thank you

**Comment by f0b04707** [comment, marked as answer]
You can setup the display [as a light](https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/docs/customization.md#set-display-as-a-light) and then, you can use homeassistant.turn_on/off on your display screen.

**EDIT**: you can use a service call to set the number of the timeout as this person mentioned https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/1060#issuecomment-1732518718

**Comment by andre1210** [comment]
Hi @WhistleMaster,
both ways are working great.
thank you for your help.
